### DISCUSSION TITLE
Export IPs within single prefix

### DISCUSSION THREAD

**Discussion by pallen182** [netbox-community/netbox#9836] [Q&A]
Hello all,

Maybe I'm missing it but is there a way to export to CSV a single prefix/subnet?

Our security team is looking to do a pentest and wants to know what IPs we have public facing and we have our public addresses onboarded and visible but can't seem to export them when inside the actual prefix. All I can find is a way to export them from the aggregate level. Any help would be greatly appreciated!

**Comment by candlerb** [comment]
This works for me:
* Go to IPAM > IP Addresses (`/ipam/ip-addresses/`)
* Select the "Filters" tab
* Enter a prefix in the "Parent prefix" field and click Search
* Click Export > Current View

This should download a file called `netbox_IP addresses.csv`

**Reply by pallen182** [reply]
Thanks! That seemed to work for me, appreciate the help :-)
